test/autoCompletionExtend.test.ts

```diff
@@ -81,7 +81,7 @@ describe('Auto Completion Tests Extended', () => {
   }
 
   function ensureExpressionSchema(): void {
-    schemaProvider.addSchema('expression', {
+    schemaProvider.addSchema('expression-schema', {
```

Comment:
I had to choose a different name for the internal test because  there was a collision with the new expression test